### ISSUE TITLE
Suggest SEO Keywords

### ISSUE THREAD

**Issue by shriyaMadan** [shriyaMadan/Youtube-Dashboard#77]
## 🔎 Feature

Suggest keywords and hashtags for increasing Search Engine Optimization(SEO) of a video.

**Comment by AbhipsaGuru1012** [comment]
Hi @shriyaMadan, can I work on this issue?

**Comment by SuNiL-Chau** [comment]
Can you elaborate @shriyaMadan ?

**Comment by shriyaMadan** [comment]
Hey @SuNiL-Chau the idea to improve SEO of the videos, we will be suggesting keywords for videos so that YouTubers can use them in the description.

**Comment by shriyaMadan** [comment]
> Hi @shriyaMadan, can I work on this issue?

yes definitely, before you start give me the idea of how you are gonna do it, also you can discuss it in discord.

**Comment by AbhipsaGuru1012** [comment]
> > Hi @shriyaMadan, can I work on this issue?
> 
> yes definitely, before you start give me the idea of how you are gonna do it, also you can discuss it in discord.

Sure, I am just pinging in the group in discord.

**Comment by Aarush-Goyal** [comment]
I would like to do that
